src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+export const initialState = [];
+
+const CHECK_STATUS = 'check-states';
```

Comment:
- [ ] Kindly make sure that your action types are following the duck pattern, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Write%20your%20book%27s%20actions%20and%20reducer%20(in%20/src/redux/books/books.js)%20using%20the%20ducks%20pattern.) requirement.

- [HINT] The duck pattern should look something like the following:
```javascript
const <ACTION_TYPE_NAME> = '<APP_NAME>/<REDUCER_NAME>/<ACTION_TYPE>';
```